src/NuGet.Core/NuGet.Packaging/Signing/VerifySignaturesResult.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Collection of signature verification results.
+    /// </summary>
+    public sealed class VerifySignaturesResult
```

Comment:
This class should have the reason for failure, if any.

Reply:
It will